doc/configuration-v3_2_experimental.md

```diff
@@ -121,9 +121,23 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_enabled_** (boolean): whether or not the service shall be enabled. When true, the service is enabled. When false, the service is disabled. When omitted, the service is unmodified. In order for this to have any effect, the unit must have an install section.
     * **_mask_** (boolean): whether or not the service shall be masked. When true, the service is masked by symlinking it to `/dev/null`.
     * **_contents_** (string): the contents of the unit.
```

Comment:
s/string/object/ I think? Also, isn't this a major schema break?

Reply:
Whoops, yeah.  This would require us to move to spec 4.0.0.  Arguably that's okay -- the change is still translatable, so we can support both 3.x and 4.0 in the same code -- but we've avoided incompatible changes so far.  But it also makes the original use case harder to implement in handwritten configs, since they'd need to convert to data URLs.